src/sql/parts/notebook/outputs/widgets.ts

```diff
@@ -0,0 +1,348 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Is this the right copyright here? Was it like this in the extension? Just checking as this looks like it's almost all Jupyter. I think we should ensure copyright reflects this.

Reply:
Yes. It is same as extension code base.